gooey/gui/widgets/widget_pack.py

```diff
@@ -48,9 +52,9 @@ def __init__(self, button_text=''):
     self.text_box = None
     self.button = None
 
-  def build(self, parent, data=None):
+  def build(self, parent, data):
     self.parent = parent
-    self.option_string = data['commands'][0] if data['commands'] else ''
+    self.option_string = self.get_command(data)

```

Comment:
this way works, though: 
```
>>> data = None
>>> data['commands'][0] if data else ''
''
```
be aware of this, though (so as it was before won't work):
```
>>> data['commands'][0] if data['commands'] else ''
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'NoneType' object is not subscriptable
```